src/main/webapp/readfile.css

```diff
@@ -0,0 +1,10 @@
+#body {
```

Comment:
You can reference content within the `<body>` element with just `body {...}`, no need to give it an ID